source/lib.ts

```diff
@@ -330,8 +332,26 @@ const rateLimit = (
 
 			// Get a unique key for the client
 			const key = await config.keyGenerator(request, response)
+
 			// Increment the client's hit counter by one.
-			const { totalHits, resetTime } = await config.store.increment(key)
+			let totalHits = 0
+			let resetTime
+			try {
+				const incrementResult = await config.store.increment(key)
+				totalHits = incrementResult.totalHits
+				resetTime = incrementResult.resetTime
+			} catch (error) {
+				if (config.passOnStoreError) {
+					console.error(
+						'express-rate-limit: error from store, allowing request without rate-limiting.',
```

Comment:
I think we should add a note informing the user what option caused this to happen. Does something like the following sound good?

```
express-rate-limit: store threw error, allowing request without rate limiting since `passOnStoreError` is true.
```

Reply:
I thought about adding an ERR_ERL_* code here, but decided against it since there could be a million different errors that all cause this line to get logged.

Reply:
Yes, good call.